nemo/collections/nlp/data/language_modeling/megatron/misc_utils.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Why not add methods to `dataset_utils.py`? Is that module already too big?

Reply:
dataset_utils.py and gpt_dataset.py had circular imports between them so I had to move the circular parts to a new file.

Reply:
I see. Perhaps change name to a more meaningful one like `base_dataset_utils` or `shared_dataset_utils`? 
`misc_utils` provides little information regarding the content.

Reply:
Sounds good! Will change the name to base_dataset_utils.